pkg/release/release.go

```diff
@@ -128,78 +128,105 @@ func (r *Release) Condition() ConditionSet {
 func (r *Release) Delete(ctx context.Context, name string) error {
 	releaseName := fmt.Sprintf("%s-%s", r.namespace, name)
 
-	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("deleting release %#q", releaseName))
```

Comment:
I removed logs from `Install` & `Delete`. With error annotations those are only changes to those functions.

Reply:
Why though? I would like to see what is happening in the e2e tests. Does the Helm client do the logging now?

Reply:
We should use `Ensure*` methods which do log. The problem I see that this methods return errors which may be handled and handling often ends up with just logging.

So then we have code like:

```
err := r.Delete(...)
if IsReleaseNotFound(err) {
log(err)
}
```

Then you have to know `Delete` does logging already, then you do some logging. Messages can change in this library and they are not aligned. Everything becomes a mess.

Reply:
I talk about informational debug logging. You talk about error logging. These are different things. I think the informational debug logging should be kept and whoever wants to do error logging can do it and having both is just fine.

Reply:
So let's take an example which can be `EnsureDeleted` method here. How should it look like? With those logs left we'd have output like:

```
deleting release `abc`
deleting release `abc`
release `abc` already deleted
```

When we have conventions we don't have to think we the method like this logs or not, because we know. Logging because good error handling is in place.

Reply:
I do not follow where the logs come from and what you want to describe.

Reply:
One `deleting release` logs from `EnsureDeleted` and second from `Delete` which is called by `EnsureDeleted`.